File: 4_File_Existance_Check/file_append.cpp
#include <iostream>
#include <fstream>

using namespace std;

int main()
{
 string str1;
cout << "File name:";
cin >> str1;
   ifstream input(str1);
  if(input) 
  {
   cout << "File exists";
  }
 else 
   cout << "File not found";
}

File: 4_File_Existance_Check/file_append.py
import os  # Import the os module to work with file system operations

# Ask the user to enter the name of the file to check
str1 = input("File Name: ")

# Check if the file exists at the given path
if os.path.exists(str1):
    print("File exists")  # If file is found, display this
else:
    print("File not found")  # If not found, inform the user

